tasks.py

```diff
@@ -281,6 +281,14 @@ def help(c):
     except:
         pass
 
+@task
+def generate_certificates(c):
+    """
+    Generates the cert.pem and cert-key.pem files
+    """
+    with c.cd(c.project_directory):
+        c.run('mkcert -cert-file ./infrastructure/docker/services/router/etc/ssl/certs/cert.pem -key-file ./infrastructure/docker/services/router/etc/ssl/certs/key.pem "*.starfleet.app"')
```

Comment:
Lets go the the right directory straight away, it will avoid duplucation of path